airflow/jobs/scheduler_job.py

```diff
@@ -1259,3 +1267,39 @@ def check_trigger_timeouts(self, session: Session = None):
         )
         if num_timed_out_tasks:
             self.log.info("Timed out %i deferred tasks without fired triggers", num_timed_out_tasks)
+
+    @provide_session
+    def _find_zombies(self, session):
+        """
+        Find zombie task instances, which are tasks haven't heartbeated for too long
+        and update the current zombie list.
+        """
+        self.log.info("Finding 'running' jobs without a recent heartbeat")
```

Comment:
Can we change this to debug or remove it entirely because seeing this every 10 seconds seems much

Reply:
Sure, changed to debug.